src/App.js

```diff
@@ -0,0 +1,12 @@
+import "./App.css";
+import Board from "./Components/Board";
+
+function App() {
+  return (
+    <div className="App">
+      <Board></Board>
```

Comment:
Components with no children can be rendered like this as well.
```suggestion
      <Board />
```